python/cuml/cluster/hdbscan/hdbscan.pyx

```diff
@@ -470,7 +472,7 @@ class HDBSCAN(Base, ClusterMixin, CMajorInputTagMixin):
         Even then in some optimized cases a tree may not be generated.
 
     """
-    sk_import_path_ = 'hdbscan'
+    _cpu_estimator_import_path = 'hdbscan.HDBSCAN'
 
     labels_ = CumlArrayDescriptor()
```

Comment:
Specifying the order of the `CumlArrayDescriptor`s would allow the automatic transfer of attributes to be more secure, especially in case the paired hdbscan library evolves.

Reply:
Can you explain what you mean by order?

Reply:
[Here is an example](https://github.com/rapidsai/cuml/blob/43f5ef3d06f77390ba773db04389ce35baafa3ee/python/cuml/linear_model/linear_regression.pyx#L258-L259) inside of the LinearRegression estimator. [It is used inside the `cpu_to_gpu` function](https://github.com/rapidsai/cuml/blob/43f5ef3d06f77390ba773db04389ce35baafa3ee/python/cuml/internals/base.pyx#L600-L605).

Reply:
@cjnolet to reaffirm, we should be fine with `order="C"` for all our descriptors, right?

Reply:
@viclafargue FWIW I think they are all 1D arrays, so it should be fine

Reply:
@divyegala yeah, just about all the intermediate and output arrays are 1d (and the order really shouldn't matter).